src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
+const HEALTH_DEFAULT = 100;
+
 class Animal {
-  // write your code here
+  constructor(name = '') {
+    this.health = HEALTH_DEFAULT;
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
+Animal.alive = [];
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name = '', hidden = false) {
+    super(name);
+    this.hidden = hidden;
+  }
+
+  hide() {
+    this.hidden = !this.hidden;
```

Comment:
Hide function shouldn't toggle the prop. Set it to `true` only

Reply:
● Animal › Herbivore should have 'hide' method which **inverts** 'hidden' property
 
 from tests